src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcShopDao.java

```diff
@@ -18,8 +18,10 @@
 
     private final Logger logger = LoggerFactory.getLogger(getClass());
     private final ShopRowMapper shopRowMapper = new ShopRowMapper();
-    final String SELECT_MANY_SHOPS_BY_IDS = "SELECT id, link_to_main_page, link_to_image, name " +
+    final String SELECT_MANY_SHOPS_BY_IDS = "SELECT id, link_to_main_page, link_to_image, name, link_to_small_image " +
             "FROM public.shops WHERE id IN ()";
```

Comment:
1) Why it is not CONSTANT? 
2) Why it named with UPPER CASE?
Please add: private static